ros/api/v1/recommendations.py

```diff
@@ -54,25 +57,22 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
                     recommendation = {}
-                    instance_price = ''
-                    summary = ''
-                    candidate_string = ''
                     rule_hit_details = rule_hit.get('details')
                     candidates = rule_hit_details.get('candidates')
                     summaries = rule_hit_details.get('summary')
-                    instance_price += f'{rule_hit_details.get("price")} {INSTANCE_PRICE_UNIT}'
+                    if rule_hit.get("key") == 'INSTANCE_IDLE':
+                        summaries = None
```

Comment:
However, I'm handling this on frontend side. If that's okay.

Reply:
Could you share what will be the value of `detected_issues` in API response?

Reply:
For IDLE state
![Screenshot from 2022-04-05 14-26-48](https://user-images.githubusercontent.com/61837065/161717992-e7fb721b-0bb6-42c1-ab59-09b0de63a105.png)

Reply:
For UNDERSIZED state
![Screenshot from 2022-04-05 14-33-39](https://user-images.githubusercontent.com/61837065/161719315-8b6d1925-74d6-4a0f-b705-c1ca0b5d0bb8.png)

Reply:
@kgaikwad good point but I am not sure if we should show `System is idle` in the `detected_issues` in case of IDLE. I feel its not an issue so we should not show it hence good with `empty array` or `None` 💯